care/facility/migrations/0319_file_upload_completed.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.2.11 on 2022-09-19 12:11
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('facility', '0318_merge_20220918_1241'),
+    ]
+
+    operations = [
+        migrations.RunSQL(
```

Comment:
can we use the orm here?

Reply:
Should be done now, Reference used: https://docs.djangoproject.com/en/4.0/ref/migration-operations/#runpython